app/controllers/nsm/claims_controller.rb

```diff
@@ -32,5 +33,17 @@ def new
         redirect_to your_nsm_claims_path, flash: { notice: t('.no_pending_claims') }
       end
     end
+
+    private
+
+    def order_and_paginate(query)
+      pagy(Sorter.call(query, @sort_by, @sort_direction))
+    end
+
+    def set_default_table_sort_options
+      default = 'date_updated'
+      @sort_by = params.fetch(:sort_by, default)
+      @sort_direction = params.fetch(:sort_direction, 'descending')
```

Comment:
sorry to be difficult... but surely the default direction got any non-date field is ascending... so..
```
    DEFAULT_SORT_FIELD = 'date_updated'
    def set_default_table_sort_options
      @sort_by = params.fetch(:sort_by, DEFAULT_SORT_FIELD)
      @sort_direction = params.fetch(:sort_direction, @sort_by == DEFAULT_SORT_FIELD ? 'descending' : 'ascending')
    end
```

Reply:
I think we're ok on this - due to how the mechanism works, any time `sort_by` is specified, `sort_direction` is also specified, so the only  time we rely on the default sort direction is when we're also relying on the default sort field, which is a date.